resources/migrations/000_migrations.yaml

```diff
@@ -6376,3 +6376,15 @@ databaseChangeLog:
             tableName: query_execution
             columnName: started_at
             newDataType: ${timestamp_type}
+
+
+# Remove fields_hash from Table model
```

Comment:
This comment just restates what the migration is doing, which is pretty clear by looking at it. It's better to have a comment that explains *why* we're doing it instead of what we're doing -- so other people know the reasoning behind it